src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -311,6 +353,31 @@ public void openBrowserAndConnect(String serviceProtocolUri, String page) {
       null
     );
   }
+
+  public void openPanel(String serviceProtocolUri, ContentManager contentManager, String tabName) {
+    String address = DevToolsUtils.generateDevToolsUrl(devtoolsHost, devtoolsPort, serviceProtocolUri, null) + "&embed=true&page=inspector";
+
+    EngineOptions options =
+      EngineOptions.newBuilder(HARDWARE_ACCELERATED).build();
+    Engine engine = Engine.newInstance(options);
```

Comment:
Are these blocking calls that take over 10ms to initialize the Engine + browser? If so this work should be done on a background thread.

Reply:
Actually, I'm still confused about this - should I put these lines into `invokeLater` as well? Or is it fine to remove `invokeLater` because all of this is on a non-main thread (verified that this is the case).

Reply:
Which thread is it on specifically? Even if it isn't the main thread it could be bad to block the thread for a couple seconds waiting for the engine to load.